Dockerfile.galaxy

```diff
@@ -0,0 +1,25 @@
+FROM quay.io/bgruening/galaxy:24.2
+
+ENV \
+GALAXY_CONFIG_TOOL_CONFIG_FILE=/qiime2_tools/qiime2_tool_conf.xml \
+GALAXY_CONFIG_CONDA_AUTO_INSTALL=true \
+GALAXY_CONFIG_CONDA_ENSURE_CHANNELS=https://packages.qiime2.org/qiime2/${EPOCH}/${DISTRO}/released,iuc,conda-forge,bioconda,defaults
```

Comment:
oh we should drop defaults from this list

Reply:
Should `--channel defaults` be dropped from the command as well?